app-text/xapers/metadata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>william@ewpettersson.se</email>
+    <name>William Pettersson</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
```

Comment:
proxy-maintainers should be last

Reply:
Not sure I understand this. proxy-maintainers is last in the actual <pkgmetadata> tag. There is a maintainer inside the <upstream> tag, for Jameson Rollins who actually runs the project, but he isn't maintaining this ebuild package.